packages/appnroll-stylelint-config/index.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  processors: ["stylelint-processor-styled-components"],
+  extends: [
+    "stylelint-config-recommended",
+    "stylelint-config-styled-components",
+  ],
+  rules: {
+    "unit-whitelist": [
+      ["%", "em", "rem", "vw", "vh", "s", "ms", "deg", "vmin", "vmax", "fr"],
+      {
+        ignoreProperties: { px: ["media-feature-name"] },
```

Comment:
why?

Reply:
As I said this config comes from mudita center. If you have any thoughts/objections please comment here.